.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Python Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  unit-tests:
+
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/project-codeflare/codeflare-sdk-precommit:v0.0.3
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install poetry
+        run: pip install poetry
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
+      - name: Install dependencies
+        run: |
+          poetry config virtualenvs.create false
```

Comment:
Does this line also need to be included for the pre-commit Containerfile and the coverage-badge workflow?

Reply:
Yes it should be, thanks for spotting that. Updated that and the poetry.lock file.